source/taskmanager/src/main/java/nl/aerius/taskmanager/TaskConsumer.java

```diff
@@ -100,8 +99,7 @@ public void messageDeliveryAborted(final Message<MessageMetaData> message, final
     taskMessageHandler.messageDeliveryAborted(message, exception);
   }
 
-  @Override
-  public void run() {
+  public void start() {
     Thread.currentThread().setName("TaskConsumer " + taskQueueName);
```

Comment:
Does it still make sense to set the thread name if this class is no longer a `Runnable`?

Reply:
No doesn't make sense. This can be removed.